<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Computer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
    <property name="spacing">
     <number>-1</number>
    </property>
    <property name="margin">
     <number>12</number>
    </property>
    <item>
     <widget class="ConsoleWidget" name="console">
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="programWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListView" name="programList"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>-1</number>
         </property>
         <item>
          <widget class="QPushButton" name="runButton">
           <property name="text">
            <string>Run Selection</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/icons.qrc">
             <normaloff>:/icons/brick_go.png</normaloff>:/icons/brick_go.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="text">
            <string>Delete</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/icons.qrc">
             <normaloff>:/icons/brick_delete.png</normaloff>:/icons/brick_delete.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionProgramList"/>
   </widget>
   <addaction name="menuProgram"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionPrint"/>
   <addaction name="actionCopy"/>
   <addaction name="separator"/>
   <addaction name="actionStop"/>
   <addaction name="actionOpenWorkingDirectory"/>
   <addaction name="actionProgramList"/>
  </widget>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../rc/icons.qrc">
     <normaloff>:/icons/cog.png</normaloff>:/icons/cog.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../rc/icons.qrc">
     <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionOpenWorkingDirectory">
   <property name="icon">
    <iconset resource="../rc/icons.qrc">
     <normaloff>:/icons/folder_magnify.png</normaloff>:/icons/folder_magnify.png</iconset>
   </property>
   <property name="text">
    <string>Open Working Directory</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../rc/icons.qrc">
     <normaloff>:/icons/printer.png</normaloff>:/icons/printer.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../rc/icons.qrc">
     <normaloff>:/icons/disk.png</normaloff>:/icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save Output</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../rc/icons.qrc">
     <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../rc/icons.qrc">
     <normaloff>:/icons/page_copy.png</normaloff>:/icons/page_copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionProgramList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../rc/icons.qrc">
     <normaloff>:/icons/application_side_list.png</normaloff>:/icons/application_side_list.png</iconset>
   </property>
   <property name="text">
    <string>Program List</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="../rc/icons.qrc">
     <normaloff>:/icons/brick_go.png</normaloff>:/icons/brick_go.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="../rc/icons.qrc">
     <normaloff>:/icons/brick_add.png</normaloff>:/icons/brick_add.png</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../rc/icons.qrc">
     <normaloff>:/icons/folder_brick.png</normaloff>:/icons/folder_brick.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConsoleWidget</class>
   <extends>QTextEdit</extends>
   <header>ConsoleWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rc/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionProgramList</sender>
   <signal>toggled(bool)</signal>
   <receiver>programWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>659</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
